rrmngmnt/network.py

```diff
@@ -545,3 +545,15 @@ def delete_interface(self, interface):
             logger.error(e)
             return False
         return True
+
+    def get_mac_by_ip(self, ip):
+        """
+        Get mac address by ip address
+
+        :param ip: ip address
+        :type ip: str
+        :return: mac address
+        :rtype: str
+        """
+        interface = self.find_int_by_ip(ip=ip)
+        return self.find_mac_by_int([interface])[0]
```

Comment:
well now you may fly on IndexError
I meant that you use an if/else statement instead of try/except